client/src/components/game.js

```diff
@@ -2,33 +2,38 @@ import { useState, useEffect } from "react";
 import QuestionCard from "./questioncard";
 
 const Game = (props) => {
+  const [questions, setQuestions] = useState([]);
+  const [score, setScore] = useState(0);
 
-    const [questions, setQuestions] = useState([]);
+  
 
-    const loadData = () => {
-        fetch('http://localhost:5000/api/game')
-            .then((response) => response.json())
-            .then(data => {
-                console.log("This is line 11", data.results);
-                setQuestions(data.results);
-            })
-    }
+  //make sure to have the backend server be consistent
+  const loadData = () => {
+    fetch("http://localhost:5000/quizgame")
```

Comment:
🔵 I'm not able to run this in github's codespaces because you hard-coded this url to localhost. This isn't a problem now, but it will be for your final project. This is one way to make it configurable: https://create-react-app.dev/docs/adding-custom-environment-variables/#what-other-env-files-can-be-used